command/acl_policy_info_test.go

```diff
@@ -31,7 +30,7 @@ func TestACLPolicyInfoCommand(t *testing.T) {
 	// Create a test ACLPolicy
 	policy := &structs.ACLPolicy{
 		Name:  "testPolicy",
-		Rules: acl.PolicyWrite,
+		Rules: "node { policy = \"read\" }",
```

Comment:
When I changed the parse error-handling to `return err` as @schmichael suggested, it caused [a test failure](https://circleci.com/gh/hashicorp/nomad/4360) because `write`, as stored here, isn’t valid HCL. Since the `Rules` aren’t actually being used anywhere in the test, I changed it to store valid HCL instead.